src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -416,25 +118,26 @@ public DeploymentOptions()
             options.IgnoreKeywordCasing = false;
             options.IgnoreSemicolonBetweenStatements = false;
 
-            PropertyInfo[] deploymentOptionsProperties = this.GetType().GetProperties();
+            // Initializing the default boolean type options to the BooleanOptionsDict
+            InitializeBooleanTypeOptions(options);
 
-            foreach (var deployOptionsProp in deploymentOptionsProperties)
+            // Excluding ExcludeObjectTypes to get the STS default values
```

Comment:
I'm a little confused with this comment. Can you explain what "Excluding ExcludeObjectTypes to get the STS default values" means? Is it getting the default values for the deployment options that are not ExcludeObjectTypes because ExcludeObjectTypes has different defaults in STS that are not the same as the DacFx defaults?

Reply:
Will update the comment clearly! 

Yes, STS has some default Excluding objects defined with the property declaration. We are considering them by explicitly not going to get the ExcludeObjectTypes from the DacFx API at this code location.